docs/resources/appdata_dsource.md

```diff
@@ -0,0 +1,176 @@
+# Resource: <resource name> delphix_appdata_dsource
+
+In Delphix terminology, a dSource is a database that the Delphix Virtualization Engine uses to create and update virtual copies of your database. 
+A dSource is created and managed by the Delphix Continuous Data Engine.
+
+
+The dSource resource allows terraform to CREATE, READ and DELETE dSources. 
+Update operation is not supported. The supported parameters are listed below.
+
```

Comment:
System Requirements
- Data Control Tower v10.0.1+ is required. Lower versions are not supported.
- The dSource Resource does not support Oracle, SQL Server, or SAP ASE. The below examples are shown from the PostgreSQL context. The parameters values can be updated for other connectors (ie AppData), such as SAP HANA, IBM Db2, etc.